share/completions/cmd.fish

```diff
@@ -0,0 +1,50 @@
+function __cmd_generate_args --description 'Function to generate args'
+  set --local current_token (commandline --current-token --cut-at-cursor)
+
+  switch $current_token
+    case '/t:*'
+      echo -e '0\tBlack
+1\tBlue
+2\tGreen
+3\tAqua
+4\tRed
+5\tPurple
+6\tYellow
+7\tWhite
+8\tGray
+9\tLight blue
+A\tLight green
+B\tLight aqua
+C\tLight red
+D\tLight purple
+E\tLight yellow
+F\tBright white' | awk -F '\t' "{ printf \"$current_token%s\t%s\n\", \$1, \$2 }"
```

Comment:
This awk pipeline is pretty gnarly! I think it could be done with Cartesian expansion instead?

Reply:
If I do smth like this:

```fish
echo prefix_(echo "1
2")
```

it prints `prefix_1 prefix_2`. The problem is dealing with embedded spaces:

```fish
echo prefix_(echo "1
2 after-space")
```

How can I print each item on a separate line to pass them as a suggested completions?

Reply:
`printf "$current_token"` is dangerous because `$current_token might contain a format specifier.

> How can I print each item on a separate line to pass them as a suggested completions?

You can use `printf %s\n` instead of `echo`.

Here's the full Cartesian product expansion:

```fish
printf %s\n $current_token(
            printf %s\t%s\n \
                1 Blue \
                2 Green
        )
```